File: C/Ex29.c
#include <stdio.h>
#include <stdlib.h>
int main(int argc, char *argv[]){
int n=0,n2=0,r, soma=0;

  for(int n=0; n != 102; n = n +2){{
        r = n2 + n;  
        printf("%i + %i = %i \n", n2, n, r);
        n2 = r;
        soma = r + soma;
      }
  }
   printf("A soma dos valores e igual a %i", soma);

  return 0;
}

File: Python/Ex29.py
n2 = 0
n = 0
soma = 0
r = 0
while(n != 102):
    r = n2 + n;  
    print(f"{n} + {n2} = {r}")
    n2 = r
    soma = soma + r
    n = n + 1
      
print(f"A soma dos valores e igual a {soma}")